core/templates/dev/head/domain/question/QuestionUpdateService.js

```diff
@@ -81,7 +81,7 @@ oppia.factory('QuestionUpdateService', [
       return diffList;
     };
 
-    var _updateContentIdsToAudioTranslations = function(newState, oldState) {
+    var _updateContentIdsInAssets = function(newState, oldState) {
```

Comment:
Hmmm, what assets does this link to exactly? It's just adding content ids to the state object right? (I assume assets would denote audio + image files linked to the exploration. I think this function does not update anything in assets, it just modifies state, so can we name it appropriately?)

Reply:
It just updates the object containing assets with the contentIds. Also currently asstes denotes Audio + text translation linked to a state. should I rename this to "modifyStateObjectHavingAssetsLinkedWithContentId"?

Reply:
That's not a very convenient name :P

Maybe updateContentIdsForAssetsInState? Is that conceptually what this function does? Or how about updateContentIdsForTranslations (assuming all translation related constructs need this function, and no others would need it)..

Also, qq... Do image assets invoke this function too? (or is it something that you plan to do? considering the general name "assets")

Reply:
> updateContentIdsForAssetsInState

Question is all about a single state, not sure whether we have to specify this over here? (but I don't think there will be an issue with this name) should I change it to 
updateContentIdsForAssetsInState?

Also, now we are considering audio and translations as a different thing (we have also changed the project name to "Audio and translation project" and these are assets for a state/question/exploration. 

> Do image assets invoke this function too?

Currently, we don't have any idea how we are going to handle image whether t will be linked via a content_id or not. This will be a part of a future project.

Reply:
Hmmm, assets seems to indicate all audio + images. maybe 
updateContentIdsForAudioAssets (Ok to drop the state part I think, as we are saying "for" and not "in")

Reply:
Written_translations are going to have text translations (not audio!), Also `
updateContentIdsForAssets` sounds more like updating contentIds not the part of state having contentIds.

Note we also have a similar function in the backend (state_domain.py Line: 1333).